src/osd/OSD.h

```diff
@@ -59,6 +59,7 @@
 #include "osd/mClockClientQueue.h"
 #include "messages/MOSDOp.h"
 #include "common/EventTrace.h"
+#include "common/numa.h"
```

Comment:
why do we need to `#include "common/numa.h"` here?

Reply:
@tchaikov 
Sort of a shortcut to not include `numa.h` in several files.
That would be a valid argument for putting the `#define cpu_set_t cpuset_t` in compat.h, and then see that is included where needed.